backend/de.metas.business/src/main/java/de/metas/order/impl/OrderBL.java

```diff
@@ -814,14 +814,8 @@ public void reserveStock(@NonNull final I_C_Order order, final org.compiere.mode
 	public void closeLine(final org.compiere.model.I_C_OrderLine orderLine)
 	{
 		Check.assumeNotNull(orderLine, "orderLine not null");
-
-		if (orderLine.getQtyDelivered().compareTo(orderLine.getQtyOrdered()) >= 0) // they delivered at least the ordered qty => nothing to do
-		{
-			return; // Do nothing
-		}
-
-		orderLine.setQtyOrdered(orderLine.getQtyDelivered());
-		InterfaceWrapperHelper.save(orderLine); // saving, just to be on the save side in case reserveStock() does a refresh or sth
+		orderLine.setIsDeliveryClosed(true);
+		InterfaceWrapperHelper.save(orderLine);
```

Comment:
@cristinamghita  cc @metas-ts ,
leave it like this for now, but for the future:

pls save it in DAO/repo